### DISCUSSION TITLE
Access via Android device

### DISCUSSION THREAD

**Discussion by jraveiro** [kimmknight/remoteapptool#112] [Ideas]
It was very interesting to develop the possibility of accessing the RemoteApp Tool with a Android device.
JRodrigues

**Comment by kimmknight** [comment]
Many people use RemoteApp Tool to serve Windows apps to Linux. This article (and video i think) might help:

https://forum.level1techs.com/t/how-to-seamlessly-run-windows-10-apps-in-a-vm-in-linux/170417

**Reply by jraveiro** [reply]
What I wanted was to use an Android device to access the Remote App installed on a Windows
